pysd/py_backend/builder.py

```diff
@@ -95,8 +104,21 @@ def time():
 
     style_file = pkg_resources.resource_filename("pysd", "py_backend/output_style.yapf")
     text = text.replace('\t', '    ')
-    text, changed = yapf.yapf_api.FormatCode(textwrap.dedent(text),
-                                             style_config=style_file)
+    try:
+        text, changed = yapf.yapf_api.FormatCode(textwrap.dedent(text),
+                                                 style_config=style_file)
+    except Exception:
```

Comment:
Do we have an issue to capture this at the moment?

Reply:
If so, lets collect everything we want to drop/break when we drop Py2 support in this milestone: https://github.com/JamesPHoughton/pysd/milestone/5